magento/magento2
# Update wikimedia/lessphp to v3

### Description (*)

Update to near stable version. Previously magento still use version 1.8.2
Valid version of this package will be >=3.0.0 <4.0.0
After this patch new version package will be 3.1.0 . From origin repo owner announcements new version support PHP8

This package magento used for server side compile less to css

Origin repo related
https://github.com/wikimedia/less.php/

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/455


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. Run static content deploy verify without issue or error
2. Preview any page in frontend or admin verify no break layout

### Questions or comments

CC @krzksz @rogyar Your comments are welcome

Should magento add new feature with official parser less without need install/depend on 3rd-party package ?
Example built-in less/sass parser by php or even we don't use php parser at all. We can use js for compilation

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#31761: Update wikimedia/lessphp to v3
